roles/load_aware/load_aware_scale_test/tasks/main.yml

```diff
@@ -1,77 +1,13 @@
-- name: Create test namespace
-  command:
-    oc create ns {{ load_aware_scale_test_namespace }}
-
-- name: Create coreutils-deps image stream
-  command:
-    oc apply -f {{ deps_imagestream }} -n {{ load_aware_scale_test_namespace }}
-
-- name: Create coreutils-make image stream
-  command:
-    oc apply -f {{ make_imagestream }} -n {{ load_aware_scale_test_namespace }}
-
-- name: Create make buildconfig
-  command:
-    oc apply -f {{ make_buildconfig }} -n {{ load_aware_scale_test_namespace }}
-
-- name: Create deps buildconfig
-  command:
-    oc apply -f {{ deps_buildconfig }} -n {{ load_aware_scale_test_namespace }}
-
-- name: Wait for coreutils-deps to finish building
-  shell:
-    oc get istag/coreutils-deps:latest -n {{ load_aware_scale_test_namespace }}
-      | awk 'NR > 1 {print $1}'
-  register: deps_image
-  delay: 15
-  retries: 80
-  until: deps_image.stdout == "coreutils-deps:latest"
-
-- name: Wait for coreutils-make to finish building
-  shell:
-    oc get istag/coreutils-make:latest -n {{ load_aware_scale_test_namespace }}
-      | awk 'NR > 1 {print $1}'
-  register: make_image
-  delay: 15
-  retries: 80
-  until: make_image.stdout == "coreutils-make:latest"
-
 - name: Generate load timeline
   shell:
     python3 {{ load_timeline_generator }} {{ load_aware_scale_test_distribution }} {{ load_aware_scale_test_duration}} {{ load_aware_scale_test_instances }} > "{{ artifact_extra_logs_dir }}/load_aware_load_timeline.log"
```

Comment:
suggestion:
- take the destination file as a parameter
- write the timeline file in YAML (lists of dicts of dicts and `yaml.dump(timeline, f)`)